customer-account-extension/src/OrderStatusBlock.liquid

```diff
@@ -0,0 +1,67 @@
+{%- if flavor contains "react" -%}
+import {
+  BlockStack,
+  reactExtension,
+  TextBlock,
+  Link,
+  Banner,
+  useApi
+} from "@shopify/ui-extensions-react/customer-account";
+
+export default reactExtension(
+  "customer-account.order-status.block.render",
+  () => <PromotionBanner />
+);
+
+function PromotionBanner() {
+  const { i18n } = useApi();
+
+  return (
+    <Banner>
+      <BlockStack inlineAlignment="center" >
+        <TextBlock>
+          {i18n.translate("earnPoints")}
+          <Link>
```

Comment:
A link without a target is weird, isn't it?

Maybe we remove the `view rewards` part?

Reply:
Where is this meant to go? Might be a good place to showcase our custom protocols like `app`, `shopify` or `extension` since not many partners know about them

Reply:
We don't have `app:`, `extension:` doesn't really make sense because we're only adding a single extension point. 😅 
`shopify:` we could do. We could link to the order list page, but with what text?

maybe @deannatron has a content idea that would give as a good excuse to put a link to host page like order list or the profile page.

Reply:
> A link without a target is weird, isn't it?
> 
> Maybe we remove the view rewards part?

Good point, this is a copy from our [public inline-extension doc](https://shopify.dev/docs/apps/build/customer-accounts/inline-extensions/build-order-status?extension=react#preview-the-extension), whose link does not have a place redirect to. We may need to update the doc as well.